### ISSUE TITLE
pred = pred.numpy().astype('uint8')此行代码执行速度过慢

### ISSUE THREAD

**Issue by wuqi930907** [PaddlePaddle/PaddleSeg#1896]
欢迎您反馈PaddleSeg使用问题，辛苦您提供以下信息，方便我们快速定位和解决问题：
 1. PaddleSeg版本：paddleseg==2.4.0
 2. PaddlePaddle版本：paddlepaddle-gpu==2.2.1.post101
 3. 操作系统信息： Ubuntu16.04
 4. Python版本号：Python3.8
 5. CUDA/cuDNN版本：CUDA10.1/cuDNN 7.6.5
 6. CPU: Intel(R) Xeon(R) CPU E5-2620 v3 @ 2.40GHz
 7. GPU: TITAN Xp
 
我在进行HarDnet模型推理时，测试了下推理速度，我修改了paddleseg/core/predict.py中第120行的代码如下
![1647866428(1)](https://user-images.githubusercontent.com/34122261/159262808-b9374180-9c1f-4ed5-8679-cb4ad44b611e.png)
发现我输入一张2048*1024大小的图片进行测试速度时，这行代码执行很慢，平均要30ms左右，请问下，有什么问题吗？
![1647866467(1)](https://user-images.githubusercontent.com/34122261/159262917-1bfd3155-1f7e-4848-af43-97cb17e6478d.png)

**Comment by wuyefeilin** [comment]
这样代码设计到从gpu传输数据到cpu， 如果tensor较大会费时较多时间

**Comment by wuqi930907** [comment]
@wuyefeilin 谢谢您的回答，我发现我换各种大小的图片去推理，logits = model(im)那一步的速度永远不变，但是pred = pred.numpy().astype('uint8')这一步的速度会随着图片大小变化，照理来讲，随着图片大小的变化，推理速度应该会变化，这是因为用的动态图推理的原因吗？因为我试验静态图那边推理比较符合常理。
